cxx-squid/src/main/java/org/sonar/cxx/preprocessor/CxxPreprocessor.java

```diff
@@ -908,6 +909,22 @@ private int matchArguments(List<Token> tokens, List<Token> arguments) {
       }
     }
 
+    // replace # with "" if sequence HASH BR occurs for body HASH __VA_ARGS__    
+    if (newTokens.size() > 3 && newTokens.get(newTokens.size() - 1).getType() == BR_RIGHT && newTokens.get(newTokens.size() - 2).getType() == HASH) {
```

Comment:
@Bertk don't know if there is really a performance difference? Would check first for # and then for comma. Think a hit for # is less likely.